sdk/aqueduct/client.py

```diff
@@ -602,6 +603,9 @@ def publish_flow(
                 behavior is 'True'.
             use_local:
                 Must be set if any artifact in the flow is derived from local data.
+            disable_snapshots:
+                If set 'True', disables snapshots for all artifacts not generating from params / metrics / checks
+                by calling `.disable_snapshot()` on each of these artifacts.
```

Comment:
This sentence seems fragmented and I am a bit confused about what it means. Pasting it to GPT gave me the following:
```
disable_snapshots:
    If set to 'True', this option disables snapshots for all artifacts that are not generated from parameters, metrics, or checks. It achieves this by calling the `.disable_snapshot()` method on each of these artifacts.
```
Is this what you're trying to say? @likawind

Reply:
Great, thank you :)